src/linear_algebra/eigenspace.lean

```diff
@@ -82,23 +83,23 @@ calc
   ... = (b • (f - b⁻¹ • am a)).ker : by rw linear_map.ker_smul _ b hb
   ... = (b • f - am a).ker : by rw [smul_sub, smul_inv_smul' hb]
 
-lemma eigenspace_eval₂_polynomial_degree_1 [field K] [vector_space K V]
+lemma eigenspace_aeval_polynomial_degree_1 [field K] [vector_space K V]
   (f : End K V) (q : polynomial K) (hq : degree q = 1) :
-  eigenspace f (- q.coeff 0 / q.leading_coeff) = (eval₂ am f q).ker :=
+  eigenspace f (- q.coeff 0 / q.leading_coeff) = (aeval f q).ker :=
 calc
   eigenspace f (- q.coeff 0 / q.leading_coeff) = (q.leading_coeff • f - am (- q.coeff 0)).ker
     : by { rw eigenspace_div, intro h, rw leading_coeff_eq_zero_iff_deg_eq_bot.1 h at hq, cases hq }
-  ... = (eval₂ am f (C q.leading_coeff * X + C (q.coeff 0))).ker
-    : by { rw C_mul', simpa [algebra_map, algebra.to_ring_hom] }
-  ... = (eval₂ am f q).ker
+  ... = (aeval f (C q.leading_coeff * X + C (q.coeff 0))).ker
+    : by { rw [C_mul', aeval_def], simpa [algebra_map, algebra.to_ring_hom], }
+  ... = (aeval f q).ker
      : by { congr, apply (eq_X_add_C_of_degree_eq_one hq).symm }
 
-lemma ker_eval₂_ring_hom'_unit_polynomial [field K] [vector_space K V]
+lemma ker_aeval_ring_hom'_unit_polynomial [field K] [vector_space K V]
   (f : End K V) (c : units (polynomial K)) :
-  ((eval₂_ring_hom' am algebra.commutes f) ↑c).ker = ⊥ :=
+  (aeval f (c : polynomial K)).ker = ⊥ :=
 begin
   rw polynomial.eq_C_of_degree_eq_zero (degree_coe_units c),
-  simp only [eval₂_ring_hom', ring_hom.of, ring_hom.coe_mk, eval₂_C],
+  simp only [aeval_def, eval₂_ring_hom', ring_hom.of, ring_hom.coe_mk, eval₂_C],
```

Comment:
You should be able to remove some of the arguments to `simp`, like `eval₂_ring_hom'`.